.github/workflows/release.yml

```diff
@@ -142,11 +142,11 @@ jobs:
           secretId: ${{ secrets.VAULT_SECRET_ID }}
       - name: "Build docker image"
         shell: bash
-        run: ./scripts/docker-release/build_docker.sh
+        run: ./scripts/docker-release/build_docker.sh "${{ env.RELEASE_VERSION_TAG }}"
       - name: "Push docker image"
         if: ${{ ! inputs.dry_run }}
         shell: bash
-        run: ./scripts/docker-release/push_docker.sh
+        run: ./scripts/docker-release/push_docker.sh "${{ env.RELEASE_VERSION_TAG }}"
```

Comment:
You'll also need to update the `push_docker` script IINM

Reply:
Aha, good catch 👼

Reply:
See https://github.com/elastic/apm-agent-java/pull/3600/commits/a0505673fbe93320347c8d129ef0bdefeba25916